vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md

```diff
@@ -1,136 +1,101 @@
+The Entando WCMS can be configured to allow or exclude any type of file extension
+to your site. For the App Builder, you should also include the MIME type for the assets you want to allow. 
 
--   **Edit:** update the information data related to every single
-    resource (image and document)
+**Default Supported Image Formats**: `jpg`, `jpeg`, `png`, `svg`, `svg+xml`
 
--   **Duplicate:** copy identically the resource from the original
+## Manage Assets:
 
--   **Download:** transfer the resource to your local machine
+1. Login to the App Builder.
+2. From the left sidebar, go to `Content` → `Assets`.
 
--   **Delete:** remove the resource
+![Digital Asset Main Page](./img/assets-manage.png) 
 
-You can list the contents by filtering with:
+* View `Images` or `Attachments` with the tabs at the top of the page.
+* Search by name or use the `Advanced filters` function to search by `Group`, `Filename` and/or `Category`. 
+* Assets are displayed and ordered by Name. `Choose View` to toggle between a list or grid format. The order can also be changed. 
+* For each resource, click on the action menu (indicated by three vertical dots) to the right to Edit, Delete and preview sizes.
 
--   **All**: All added files are displayed, images and documents both.
+![Digital Asset Information](./img/assets-info.png)
 
--   **Images**: All added images are displayed
+### Add a New Asset
+1. From the Content Assets page in the App Builder, click `Add` on the right to add a new image or document file. 
 
--   **Documents**: All added documents are displayed. 
+![Add Digital Assets Window](./img/assets-add.png)
 
-## Editing Assets
-Once you have a digital asset in place, you have several editing and viewing options. Click the asset's *Actions* button and select **Edit** to update the asset's name/description, categories, and the file represented.
+2. Group: choose from the drop-down menu. This is a required field and defaults to `Free Access`. Additional groups can be defined in the Users section.  
+3. Categories: add your asset to the appropriate category with the `+` symbol under `Join`. This will be used in searching and sorting. This field is not mandatory. New Categories can be created in `Content` → `Categories`.
+4. Drag and Drop your files into the window or click `Browse`. The file you choose will appear in the Name field. You can choose multiple files at one time. You can also use the `Add Another Resource` button and `Choose file` individually.
+5. After all files have been chosen, click the `Add` button to upload the files. This takes you back to the main assets page where your uploaded files will be displayed in the table. If you have an image file, see Edit Assets below. 
```

Comment:
" are displayed" (present tense)

maybe "...Edit Assets section" just to be clear?